plugin.go

```diff
@@ -198,7 +199,7 @@ func (p *Plugin) runJobs() {
 }
 
 func debug(format string, args ...interface{}) {
-	if os.Getenv("DEBUG") != "" {
+	if os.Getenv("DEBUG") != "" || os.Getenv("DEBUG") != "" {
```

Comment:
This change doesn't make sense.

Reply:
wait what?
I've added os.Getenv("VERBOSE")
obviously it doesn't make sense like that